octobot_trading/exchanges/parser/positions_parser.py

```diff
@@ -332,11 +349,30 @@ def timestamp_not_found():
         )
 
 
-# keep keys here from exchanges that are 100% safe
-BYBIT_BANKRUPTCY_PRICE = "bust_price"
-BYBIT_REALIZED_PNL = "cum_realized_pnl"
-BYBIT_CLOSING_FEE = "occ_closing_fee"
-BYBIT_VALUE = "position_value"
+# only keep keys here from exchanges that are 100% safe with any exchange
+class LiquidationSynonyms:
+    keys = ["bust_price"]
+
+
+class RealizedPnlSynonyms:
+    keys = ["cum_realized_pnl"]
+
+
+class ClosingFeeSynonyms:
+    keys = ["occ_closing_fee"]
+
+
+class ValueSynonyms:
+    keys = ["position_value"]
+
+
+class SizeSynonyms:
+    keys = ["positionAmt", ExchangeCols.CONTRACTS.value]
```

Comment:
I don't understand the idea behind `Synonyms`. Why not just letting the tentacle set this value instead of trying "synonyms"?

Reply:
so we dont need to set the same thing twice ever again. Maybe some other exchanges use the same wording.
there are 100+ exchanges on ccxt, we have to be as generic as possible to save time

Reply:
also it will first try the common way and only if it fails it tries the others